src/Command/ProjectBuildCommand.php

```diff
@@ -2,311 +2,320 @@
+          'to' => 'https://www.' . $site['url'] . '/',
+        ];
+      }
+
+      // If a site folder doesn't exist under project/sites, create it and
+      // provide a settings file.
+      if (!$this->fs()->exists('/project/sites/' . $site_id)) {
+        $io->text('Creating a site directory for ' . $site_id . ' under project/sites/');
+        $this->fs()->mkdir('/project/sites/' . $site_id);
+        $this->fs()->copy('/.lando/config/multisite.settings.php', '/project/sites/' . $site_id . '/settings.php');
+      }
+
+      // Enable our multisite entry by linking from the sites directory to
+      // the project directory.
+      try {
+        $this->fs()->symlink('//app/project/sites/' . $site_id, '/web/sites/' . $site_id);
+        $io->text('Linking sites directory');
+      }
+      catch (IOExceptionInterface $exception) {
+        $io->error("An error occurred while linking $site_id site directory: " . $exception->getMessage());
+      }
+
+      // If a site config doesn't exist under project/config, create it.
+      if (!$this->fs()->exists('/project/config/' . $site_id)) {
+        $io->text('Creating config directory for ' . $site_id . ' under project/config/');
+        $this->fs()->mkdir('/project/config/' . $site_id);
+        $this->fs()->dumpFile('/project/config/' . $site_id . '/.gitkeep', "");
+
+        // Create the default config directories if they don't already exist.
+        foreach (['config', 'hosted', 'local', 'production'] as $directory) {
+          $io->text('Creating default config directories');
+          if (!!$this->fs()->exists('/project/config/' . $site_id . '/config/' . $directory)) {
+            $this->createDirectory('/project/config/' . $site_id . '/config/' . $directory);
+          }
         }
+      }
+    }
 
-        // Check for an .env file and copy example if missing
-        if (!$filesystem->exists(getcwd() .'/.env')) {
-            try {
-                $filesystem->copy(getcwd() . '/.env.sample', getcwd() . '/.env');
-                $io->success('Created local .env file');
-            }
-            catch (IOExceptionInterface $exception) {
-                $io->error('Unable to create local .env file, error: ' . $exception->getMessage());
-            }
-        }
+    // Update platform post deploy hook with list of deployed sites.
+    $io->text('Updating Platform post-deploy hook');
+    $platform['hooks']['post_deploy'] = str_replace('<deployed_sites_placeholder>', implode(' ', $deployed_sites), $platform['hooks']['post_deploy']);
+
+    // Add 'Catch all' to PlatformSH routing.
+    $io->text("Adding 'Catch all' to Platform routes.");
+    $platform_routes['https://www.{all}/'] = [
+      'type' => 'upstream',
+      'upstream' => $platform['name'] . ':http',
+      'cache' => [
+        'enabled' => 'false',
+      ],
+    ];
+
+    $platform_routes['https://{all}/'] = [
+      'type' => 'redirect',
+      'to' => 'https://www.{all}/',
+    ];
```

Comment:
Just a thought, if we have manually removed a site route from .platform/routes.yaml because teh site has gone live, will this code reinstate it ?

Reply:
If the site is live it should have the deploy option in project.yml set to true in which case it won't have an entry in the routes file and it'll be caught by the 'https://{all}/' definition.